core/loadpoint.go

```diff
@@ -1450,6 +1464,39 @@ func (lp *LoadPoint) Update(sitePower float64, cheap bool, batteryBuffered bool)
 		err = lp.setLimit(targetCurrent, required)
 	}
 
+	// reset WakeUpCalled then car is charging again
+	if lp.status == api.StatusC && (lp.wakeUpTimer.called || lp.wakeUpTimer.active) {
+		lp.log.DEBUG.Printf("reset WakeUpTimer - charing - Active:%ds", lp.wakeUpTimer.duration())
+		lp.wakeUpTimer.Reset()
+	}
+	// WakeUp checks
+	if lp.enabled && mode != api.ModeOff && lp.chargeCurrent > 0 && lp.vehicleSoc < 99 && lp.status == api.StatusB {
+		if lp.wakeUpTimer.active {
```

Comment:
Kapseln. Als funktion oder im timer

Reply:
30s als konstante, idealerweise im timer

Reply:
ja kann ich in den Timer verschieben - würde ihn aber dann umbenennen in WakeUpTimer da er dann spezifisch nur dafür ist ok?

Reply:
kann das eigentlich passieren?
!lp.enabled && lp.chargeCurrent > 0?
oder
mode == api.ModeOff && lp.chargeCurrent > 0?
oder
lp.enabled && mode == api.ModeOff?
eigentlich darf der lp.chargeCurrent nur größer 0 sein wenn (lp.enabled && mode != api.ModeOff) zutrifft oder?

Reply:
`lp.enabled && lp.status == api.StatusB && lp.vehicleSoc < 100` sollte IMHO völlig ausreichend sein.

Reply:
aber soc kleiner 99 da ich es schon öfter hatte das er bei 99% nicht mehr laden will - weiß nicht ob es daran liegt das ein Hybrid ist und der Unterschied zwischen 99 und 100 so klein ist??? aber kann ich auch auf 100 ändern dann versucht er ihn einmal zu wecken. Bei anderen macht es bestimmt wieder sinn da 1% fast 1kWh sein kann.